lua/dap-python.lua

```diff
@@ -2,6 +2,9 @@ local api = vim.api
 local M = {}
 
 
+M.test_runner = 'unittest'
```

Comment:
Do you think it should default to `pytest` if there is a `pytest` executable available? Afaik the runner should be able to run `unittest` style tests too?

Reply:
ok, `vim.fn.executable('pytest') == 1 and 'pytest' or 'unittest'`

Reply:
But then it should be able to run classes.

Reply:
Ah right, sorry. Can you revert it for now? I'll then merge this - can be followed up later.

Reply:
@mfussenegger don't you think I should revert that change until nvim-dap-python properly supports the detection of test classes versus normal ones?

Reply:
> don't you think I should revert that change until nvim-dap-python properly supports the detection of test classes versus normal ones?

Yes that was what I meant.

Reply:
done